CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+The format is based on
+[Keep a Changelog](http://keepachangelog.com)
+& makes a strong effort to adhere to
+[Semantic Versioning](http://semver.org).
+
+Tracking in this Changelog began for this project in version 0.8.4.
+If you're looking for changes from before this, refer to the project's
+git logs & PR history.
+
+The headers used in [Keep a Changelog](http://keepachangelog.com) are:
+
+- Added - for new features.
+- Changed - for changes in existing functionality.
+- Deprecated - for soon-to-be removed features.
+- Removed - for now removed features.
+- Fixed - for any bug fixes.
+- Security - in case of vulnerabilities.
+
+# [Unreleased](https://github.com/puppetlabs/beaker/compare/0.8.4...master)
```

Comment:
Perhaps this shouldn't be added but rather when the actual release gets made? Because if you now tag 0.8.4 and release it, it'll contain a changelog with an `Unreleased` entry.

Reply:
Yeah, I just copied it over from `beaker`. It's nice to have a link to what's coming up IMO

Reply:
Yes, but as for the workflow, I think what you end up with is:
* Prepare PR to release x.y.z
* Merge PR
* Create tag
* Push tag
* Release tag
* Prepare branch for next release

This is the first step of that and adding `Unreleased` feels like part of the last step. Does that make sense?